serviceWrappers/linux/kLiveController

```diff
@@ -21,48 +21,71 @@
 ###
 # extend callable functions from "functions" script
 ###
+NAME=kLiveController
+LOG_DIR=/var/log
```

Comment:
Sorry but I don't understand. What's so hard about:
```
if [ -z "$LIVE_LOG_DIR" ];then
   LIVE_LOG_DIR=/var/log
fi
```
This way, if LIVE_LOG_DIR was defined OUTSIDE the script, it shall be used, otherwise /var/log will be used.
This solution will work for both OP and SaaS and will require no replacements inside init or other files.